content/documentation/pages/6-recipes/2-file-ingest/1-sftp-to-jdbc.md

```diff
@@ -0,0 +1,936 @@
+
+You can [download the project](https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/recipes/file-ingest/file-to-jdbc/file-to-jdbc.zip?raw=true) that contains the source code and sample data from your browser, or from the command line:
+
+```bash
+wget https://github.com/spring-cloud/spring-cloud-dataflow-samples/blob/master/dataflow-website/recipes/file-ingest/file-to-jdbc/file-to-jdbc.zip?raw=true -O file-to-jdbc.zip
+```
+
+<!--TIP-->
+
+If you choose not to build the task application yourself, the executable jar is published to the [Spring Maven repository](https://repo.spring.io/libs-snapshot-local/io/spring/cloud/dataflow/ingest/ingest/1.0.0.BUILD-SNAPSHOT/) and to the [springcloud/ingest](https://hub.docker.com/r/springcloud/ingest/) Docker repository.
+
+<!--END_TIP-->
+
+The pipeline is built using the following pre-packaged Spring Cloud Stream applications:
+
+- [sftp-dataflow-source](https://github.com/spring-cloud-stream-app-starters/sftp/tree/master/spring-cloud-starter-stream-source-sftp-dataflow) an SFTP source configured to emit a Task Launch Request whenever it detects a new file in one or more polled SFTP directories.
+- [dataflow-task-launcher-sink](https://github.com/spring-cloud-stream-app-starters/tasklauncher-dataflow/tree/master/spring-cloud-starter-stream-sink-task-launcher-dataflow) a sink that acts as a REST client to the Data Flow server to launch a Data Flow task.
+
+This pipeline runs on all supported Data Flow platforms.
+The SFTP source downloads each file from the SFTP server to a local directory before sending the task launch request.
+The request sets `localFilePath` as a command line argument for the task. When running on a cloud platform, we need to mount a shared directory available to the SFTP source container and the task container.
+For this example, we will set up an NFS mounted directory.
+Configuring the environment and containers for NFS is platform specific and is described here for Cloud Foundry v2.3+ and minikube.
+
+## Prerequisites
+
+### Data Flow Installation
+
+Make sure you have installed Spring Cloud Data Flow to the platform of your choice:
+
+- [Local](%currentPath%/installation/local/)
+- [Cloud Foundry](%currentPath%/installation/cloudfoundry)
+- [Kubernetes](%currentPath%/installation/kubernetes/)
+
+<!--NOTE-->
+
+**NOTE**: For kubernetes, the sample task application is configured to use `mysql`. The Data Flow server must also be configured for mysql.
+
+**TODO**: Support for Postgres
```

Comment:
We can probably wrap this in a comment as opposed to showing as text in the generated content.